docs/literate/src/files/first_steps/changing_trixi.jl

```diff
@@ -0,0 +1,77 @@
+#src # Changing Trixi.jl itself
+
+# If you plan on editing Trixi.jl itself, you can download Trixi.jl locally and run it from
+# the cloned directory.
+
+
+# ## Cloning Trixi.jl
+
+
+# ### Windows
+
+# If you are using Windows, you can clone Trixi.jl by using the GitHub Desktop tool:
+# - If you do not have a GitHub account yet, create it on
+#   the [GitHub website](https://github.com/join).
+# - Download and install [GitHub Desktop](https://desktop.github.com/) and then log in to
+#   your account.
+# - Open GitHub Desktop, press `Ctrl+Shift+O`.
+# - In the opened window, paste `trixi-framework/Trixi.jl` and choose the path to the folder where
+#   you want to save Trixi.jl. Then click `Clone` and Trixi.jl will be cloned to your computer. 
+
+# Now you cloned Trixi.jl and only need to tell Julia to use the local clone as the package sources:
+# - Open a terminal using `Win+R` and `cmd`. Navigate to the folder with the cloned Trixi.jl using `cd`.
```

Comment:
```suggestion
# - Open a terminal using `Win+r` and `cmd`. Navigate to the folder with the cloned Trixi.jl using `cd`.
```
In the `getting_started.jl` part of the tutorial you wrote that one should use `Win+r` for this step.